### PR TITLE
[10.0][ADD] medical_pathology_import: Import interface

### PR DESCRIPTION
**Pull request by obulkin**
[ADD] ``medical_pathology_import``: Import interface
* Add wizard that provides UI for importing external pathology data. The wizard methods effectively define an interface that must be extended to support specific import use cases

This PR is a continuation of https://github.com/OCA/vertical-medical/pull/140, which was closed in favor of a v10 submission.

Depends on:
- [x] https://github.com/OCA/vertical-medical/pull/190

### CODE REVIEW

File: medical_pathology_import/wizards/medical_pathology_import.py [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+            method()
+        except AttributeError:
+            self.zip_uri = False
+            self.file_name = False
+
+    @api.model
+    def create(self, vals):
+        """ It begins import process after create """
+        record = super(MedicalPathologyImport, self).create(vals)
+        record.do_import()
```
Comment (lasley) [inline medical_pathology_import/wizards/medical_pathology_import.py]: Let's add a context key to skip this. `no_import=True`

Never know when you need to skip a long operation when coding.

File: medical_pathology_import/wizards/medical_pathology_import.py [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+        return 'medical_pathology_category_%s' % name
+
+    @api.multi
+    def _upsert_pathology(
+        self, name, code, category, code_type, id_pref, parent=None, note=None,
+    ):
+        """ It updates or creates a new Pathology for arguments.
+
+        Args:
+            name: (str) Name of pathology to create.
```
Comment (lasley) [inline medical_pathology_import/wizards/medical_pathology_import.py]: Ewww I did these wrong. Can you please fix? Format is:

```
variable_name (type): Description
```

File: medical_pathology_import/wizards/medical_pathology_import.py [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+            code: (str) Code for pathology.
+            category: (medical.pathology.category) Singleton representing the
```
Comment (lasley) [inline medical_pathology_import/wizards/medical_pathology_import.py]: Models should be the actual name and not the `_name`. So `MedicalPathologyCategory`

File: medical_pathology_import/wizards/medical_pathology_import.py [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+                category for the new pathology.
+            code_type: (medical.pathology.code.type) Singleton representing
+                the code type of the pathology (ICD-10-CM).
+            id_pref: (str) External ID prefix for pathology
+            parent: (medical.pathology) Parent pathology.
+            note: (str) Pathology notes.
+        Returns:
+            (medical.pathology) Newly created Pathology.
```
Comment (lasley) [inline medical_pathology_import/wizards/medical_pathology_import.py]: `type: Description`

**Review by lasley — CHANGES_REQUESTED** [review]
Thanks @obulkin - few minor changes

**Comment by obulkin** [comment]
@lasley - I've pushed up a commit that should address your comments.

**Review by lasley — APPROVED** [review]

**Comment by lasley** [comment]
Merging - there are no other reviewers for this branch

**Pull request merged**